apps/v1/types.go

```diff
@@ -24,6 +24,9 @@ import (
 // Triggers can be disabled to allow manual control over a deployment. The "strategy" determines how the deployment
 // is carried out and may be changed at any time. The `latestVersion` field is updated when a new deployment
 // is triggered by any means.
+//
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
```

Comment:
where is this sentence from? What should it tell a user? "within a major release"

Reply:
The compatibility docs define major, minor, and z-stream (_major_._minor_._z-stream_) releases.